app/src/main/java/swati4star/createpdf/util/lambda/Consumer.java

```diff
@@ -0,0 +1,17 @@
+package swati4star.createpdf.util.lambda;
+
+/**
+ * A custom Implementation of {@link java.util.function.Consumer}.
+ * Used because API Level 24 is required to use the java variant.
+ * @param <T> the type of the input to the operation
+ */
+public interface Consumer<T> {
```

Comment:
Is this the apt name for this interface? 
and update the comment also :)

Reply:
what is an "apt name"?

Reply:
FunctionTemplate

Reply:
The java Functional interface is called Consumer. I didn't find a specific name for a c++ function template with that purpose.

Reply:
Let's leave it as it then!

Reply:
ok